src/pocketmine/command/Command.php

```diff
@@ -239,19 +239,43 @@ public function getPermissionMessage() : ?string{
 	}
 
 	/**
+	 * Returns the generic description message of this command.
+	 *
 	 * @return string
 	 */
 	public function getDescription() : string{
 		return $this->description;
 	}
 
 	/**
+	 * Returns the specific description message of this command for the CommandSender at this moment.
```

Comment:
I think this is misleading. This doesn't apply for players - they'll see the message that was there when the command data was sent (i.e. during login).